pkg/rbac/userData.go

```diff
@@ -176,23 +184,41 @@ func (user *userData) getNamespacedResources(cache *Cache, ctx context.Context,
 		if err != nil {
 			klog.Error("Error creating SelfSubjectRulesReviews for namespace", err, ns)
 		} else {
-			klog.V(9).Infof("TokenReview Kube API result: %v\n", prettyPrint(result.Status))
+			klog.V(9).Infof("SelfSubjectRulesReviews Kube API result: %v\n", prettyPrint(result.Status))
 		}
-		for _, rules := range result.Status.ResourceRules { //iterate objects
+		for _, rules := range result.Status.ResourceRules {
 			for _, verb := range rules.Verbs {
-				if verb == "list" || verb == "*" { //drill down to list only
+				if verb == "list" || verb == "*" {
 					for _, res := range rules.Resources {
 						for _, api := range rules.APIGroups {
 							user.nsResources[ns] = append(user.nsResources[ns], resource{apigroup: api, kind: res})
 						}
 					}
 				}
```

Comment:
This will need to be optimized to account for `*`, but let's merge this PR first and update this right after.